packages/kit-headless/src/components/checkbox/checkbox.test.ts

```diff
@@ -28,423 +28,408 @@ async function setup(page: Page, exampleName: string) {
   };
 }
 
-test.describe('render behavior', () => {
-  test.describe('checklist', () => {
-    test(`GIVEN a mixed checklist
-        WHEN the checklist renders
-        IT should render the mixed img
-        AND not the true img`, async ({ page }) => {
-      const exampleName = 'test-controlled-list-mixed';
-      await setup(page, exampleName);
-      await expect(page.locator('#mixed-img')).toBeVisible();
-      await expect(page.locator('#true-img')).toBeHidden();
-    });
-
-    test(`GIVEN an all-checked checklist
-        WHEN the checklist renders
-        IT should  render the true img
-        AND not the mixed img`, async ({ page }) => {
-      const exampleName = 'test-controlled-list-true';
-      await setup(page, exampleName);
-      await expect(page.locator('#true-img')).toBeVisible();
-      await expect(page.locator('#mixed-img')).toBeHidden();
-    });
+test.describe('checklist', () => {
+  test(`GIVEN a mixed checklist
+  WHEN the checklist renders
+  IT should render the mixed img
```

Comment:
@TheMcnafaha I think we should be consistent about our test naming conventions.

So my OCD is forcing me to say that it should be `THEN` instead of `IT` 😅

it could be `THEN render the mixed image` or something like that (the `should` keyword isn't necessary most of the time)

Reply:
I agree. I'll make the naming changes and merge.